docs/guides/mozilla-sops.md

```diff
@@ -131,10 +131,24 @@ AWS IAM Role example:
 }
 ```
 
+Azure:
+
 When using Azure Key Vault you need to authenticate the kustomize controller either by passing
 [Service Principal credentials as environment variables](https://github.com/mozilla/sops#encrypting-using-azure-key-vault)
 or with [add-pod-identity](https://github.com/Azure/aad-pod-identity).
 
+GCP Role:
+
+When using GCP, create a service account with the role `Cloud KMS CryptoKey Encrypter/Decrypter`, create an bind an IAM Rolebinding 
+between the GCP service account to the `default` service account of the `flux-system`, and annotate the service account
+with the GCP service account.
+```sh
+kubectl annotate serviceaccount \
```

Comment:
Do users need to restart kustomize-controller pod after this?

Reply:
No, they don't

Reply:
No, they don't